packages/Liquid-Tests.package/ObjectRepoTests.class/instance/setUpRemoteServer.st

```diff
@@ -1,7 +1,11 @@
 tests
 setUpRemoteServer
-	|port url|
-	port := 10000 random.
-	server := RemoteRepoServer new listenOn: port.
+	|port url canProceed|
+	canProceed := false.
+	[canProceed] whileFalse: [
+		canProceed := true.
+		port := (10000 to: 50000) atRandom.
+		[server := (RemoteRepoServer new listenOn: port)] ifError: [canProceed := false].
```

Comment:
[pebble] This could potentially run forever, and I foresee it becoming a future headache. I'll open a separate PR with the global increment-thing in the future.